ui/components/app/menu-droppo.js

```diff
@@ -76,6 +81,7 @@ export default class MenuDroppoComponent extends Component {
       <div
         style={baseStyle}
         className={`menu-droppo-container ${containerClassName}`}
+        data-testid={`${containerClassName}`}
```

Comment:
I think this can just be:

`data-testid={containerClassName}
 `

Reply:
Resolved in [65ee488](https://github.com/MetaMask/metamask-extension/commit/65ee488802fe9944f4b3af48014315d1f9b8916e)